04-conditional-logic/factors.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def get_a_positive_integer_from_user():
+    response = int(input("Enter a positive integer: "))
```

Comment:
Not in the boundaries of this exercise but bear in mind that the user might enter a text instead of a number, in which case the casting to int will fail with an exception.